### PR TITLE
Print confirmation message to user when a book status has changed inc…

### PR DESCRIPTION
**Pull request by fermolanoc**
@saida93522 This commit includes a message to user confirming when a book status has changed. If a book has been read or not, a confirmation message will tell the user what's the book read status including the book title and book author name so user can be sure the right book was modified. This solves issue #6

### CODE REVIEW

**Comment by saida93522** [comment]
It looks great, much more user-friendly. ready to merge.

File: main.py [hunk 1]
```diff
@@ -76,6 +76,12 @@ def change_read():
     if book != None:
         new_read = ui.get_read_value()
         book.read = new_read
+        if new_read == True:
```
Comment (claraj) [inline main.py:79]: Or you could say
`if new_read:`

**Review by claraj — COMMENTED** [review]
You can be more concise when comparing to True or False.

**Comment by claraj** [comment]
Looks good!

**Pull request merged**